plugins/obs-ffmpeg/obs-ffmpeg-nvenc.c

```diff
@@ -183,11 +189,22 @@ static bool nvenc_update(void *data, obs_data_t *settings)
 		enc->context->rc_min_rate = bitrate * 1000;
 		cqp = 0;
 	}
-
+	
+	if (rclookahead > 0)
```

Comment:
Instead of adding this `if` block here, I think it would be better placed below the block of `av_opt_set_int` calls below.